### PR TITLE
Add new commands to Ews extension

### PR DESCRIPTION
**Pull request by merit-maita**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by thefrieddan1 — APPROVED** [review]

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml [hunk 1]
```diff
@@ -610,7 +610,7 @@ script:
...
       name: to_ip
-    - description: The Message-ID header field of the message by which to filter the results. This value is also known as the Client ID. The format of the Message-ID depends on the messaging server that sent the message. The value should be unique for each message. However, not all messaging servers create values for the Message-ID in the same way. Be sure to include the full Message ID string (which may include angle brackets) and enclose the value in quotation marks (for example,"d9683b4c-127b-413a-ae2e-fa7dfb32c69d@DM3NAM06BG401.Eop-nam06.prod.protection.outlook.com").        '
+    - description: The Message-ID header field of the message by which to filter the results. This value is also known as the Client ID. The format of the Message-ID depends on the messaging server that sent the message. The value should be unique for each message. However, not all messaging servers create values for the Message-ID in the same way. Be sure to include the full Message ID string (which may include angle brackets) and enclose the value in quotation marks (for example,"d9683b4c-127b-413a-ae2e-fa7dfb32c69d@DM3NAM06BG401.Eop-nam06.prod.protection.outlook.com").        '.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml:613]:
```suggestion
    - description: The Message-ID header field of the message by which to filter the results. This value is also known as the Client ID. The format of the Message-ID depends on the messaging server that sent the message. The value should be unique for each message. However, not all messaging servers create values for the Message-ID in the same way. Be sure to include the full Message ID string (which may include angle brackets) and enclose the value in quotation marks (for example,"d9683b4c-127b-413a-ae2e-fa7dfb32c69d@DM3NAM06BG401.Eop-nam06.prod.protection.outlook.com").
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml [hunk 2]
```diff
@@ -2034,30 +2034,92 @@ script:
...
+      description: The priorty of the rule.
+      type: string
+    - contextPath: EWS.Rule.Description
+      description: The description of the rule.
+      type: string
+    - contextPath: EWS.Rule.StopProcessingRules
+      description: Stop Processing Rules.
+      type: string
+    - contextPath: EWS.Rule.IsValid
+      description: wheather the rule is valid or not.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml:2108]:
```suggestion
      description: Whether the rule is valid or not.
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml [hunk 3]
```diff
@@ -2034,30 +2034,92 @@ script:
...
+      required: true
+    outputs:
+    - contextPath: EWS.Rule.Rule
+      description: The rule identity.
+      type: string
+    - contextPath: EWS.Rule.RuleName
+      description: The rule name.
+      type: string
+    - contextPath: EWS.Rule.IsEnabled
+      description: Wheather the rule is enabled or not.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml:2096]:
```suggestion
      description: Whether the rule is enabled or not.
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml [hunk 4]
```diff
@@ -2034,30 +2034,92 @@ script:
...
+      name: limit
+    outputs:
+      - contextPath: EWS.Rule.RuleIdentity
+        description: The rule identity.
+        type: string
+      - contextPath: EWS.Rule.Name
+        description: The rule name.
+        type: string
+      - contextPath: EWS.Rule.Enabled
+        description: Wheather the rule is enabled or not.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml:2074]:
```suggestion
        description: Whether the rule is enabled or not.
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md [hunk 5]
```diff
@@ -1427,3 +1427,90 @@ Retrieves information about the AuditBypassEnabled property value for user accou
...
+| mailbox | The mailbox that contains the Inbox rule. | Required | 
+| limit | Maximum number of rules to get. A value of 0 means to get all rules | Optional | 
+
+
+#### Context Output
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| EWS.Rule.RuleIdentity | String | The rule identity. | 
+| EWS.Rule.Name | String | The rule name. | 
+| EWS.Rule.Enabled | Boolean| If the rules is enabled or not. |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md:1451]:
```suggestion
| EWS.Rule.Enabled | Boolean| Whether the rule is enabled or not. | 
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md [hunk 6]
```diff
@@ -1427,3 +1427,90 @@ Retrieves information about the AuditBypassEnabled property value for user accou
...
+| --- | --- | --- |
+| mailbox | The mailbox that contains the Inbox rule. | Required | 
+| identity | the id of the rule. | Required |
+
+#### Context Output
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| EWS.Rule.Rule | String | The rule identity. | 
+| EWS.Rule.RuleName | String | The rule name. | 
+| EWS.Rule.IsEnabled | Boolean| If the rules is enabled or not. |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md:1482]:
```suggestion
| EWS.Rule.IsEnabled | Boolean| Whether the rule is enabled or not. | 
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md [hunk 7]
```diff
@@ -1427,3 +1427,90 @@ Retrieves information about the AuditBypassEnabled property value for user accou
...
+
+#### Base Command
+
+`ews-get-rule`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| mailbox | The mailbox that contains the Inbox rule. | Required | 
+| identity | the id of the rule. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md:1475]:
```suggestion
| identity | The ID of the rule. | Required |
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md [hunk 8]
```diff
@@ -1427,3 +1427,90 @@ Retrieves information about the AuditBypassEnabled property value for user accou
...
+
+#### Context Output
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| EWS.Rule.Rule | String | The rule identity. | 
+| EWS.Rule.RuleName | String | The rule name. | 
+| EWS.Rule.IsEnabled | Boolean| If the rules is enabled or not. | 
+| EWS.Rule.Priority | String | the rule priority. |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md:1483]:
```suggestion
| EWS.Rule.Priority | String | The rule priority. |
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md [hunk 9]
```diff
@@ -1427,3 +1427,90 @@ Retrieves information about the AuditBypassEnabled property value for user accou
...
+| EWS.Rule.Description | String | The description of the rule. | 
+| EWS.Rule.StopProcessingRules | Boolean| If to stop processing the rule or not. |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md:1485]:
```suggestion
| EWS.Rule.StopProcessingRules | Boolean| Whether to stop processing the rule or not. | 
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md [hunk 10]
```diff
@@ -1427,3 +1427,90 @@ Retrieves information about the AuditBypassEnabled property value for user accou
...
+#### Context Output
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| EWS.Rule.Rule | String | The rule identity. | 
+| EWS.Rule.RuleName | String | The rule name. | 
+| EWS.Rule.IsEnabled | Boolean| If the rules is enabled or not. | 
+| EWS.Rule.Priority | String | the rule priority. |
+| EWS.Rule.Description | String | The description of the rule. | 
+| EWS.Rule.StopProcessingRules | Boolean| If to stop processing the rule or not. | 
+| EWS.Rule.IsValid | Boolean | If the rule is valid or not. |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md:1486]:
```suggestion
| EWS.Rule.IsValid | Boolean | Whether the rule is valid or not. | 
```

File: Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md [hunk 11]
```diff
@@ -1427,3 +1427,90 @@ Retrieves information about the AuditBypassEnabled property value for user accou
...
+
+#### Base Command
+
+`ews-remove-rule`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| mailbox | The mailbox that contains the Inbox rule. | Required | 
+| identity | the id of the rule. | Required |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/README.md:1508]:
```suggestion
| identity | The ID of the rule. | Required | 
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Review by thefrieddan1 — APPROVED** [review]

**Pull request closed without merging**